dask/array/core.py

```diff
@@ -782,6 +784,9 @@ def finalize(results):
 class Array(Base):
     """ Parallel Array
 
+    Do not construct Dask.arrays with this class.  Instead use
+    ``da.from_array``.
```

Comment:
Perhaps "Construct a `da.Array`. Not user facing, recommend using `da.from_array`." Sometime it can be useful to use the constructor, but you should know what you're doing at that point.